$NetBSD: patch-ae,v 1.3 2009/09/13 16:40:22 tron Exp $

Build for for Mac OS X Snow Leopard taken from Mac Ports:

http://trac.macports.org/browser/trunk/dports/sysutils/cdrtools/files/patch-libscg__scsi-mac-iokit.c?rev=56542

--- libscg/scsi-mac-iokit.c.orig	2009-09-03 21:47:36.000000000 +0100
+++ libscg/scsi-mac-iokit.c	2009-09-13 17:34:22.000000000 +0100
@@ -59,7 +59,7 @@
 #include <Carbon/Carbon.h>
 #include <IOKit/IOKitLib.h>
 #include <IOKit/IOCFPlugIn.h>
-#include <IOKit/scsi-commands/SCSITaskLib.h>
+#include <IOKit/scsi/SCSITaskLib.h>
 #include <mach/mach_error.h>
 
 struct scg_if {
